compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -185,6 +185,12 @@ def WorkGroupSwizzle :
   ];
 }
 
+def PadDynamicAlloc :
+    Pass<"iree-pad-dynamic-alloc", "func::FuncOp"> {
```

Comment:
To be consistent with other passes here, could you name this `iree-codegen-pad-dynamic-alloc`?